env/app_vars.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/sh
+>.env
+echo "# Vault Secrets" >>.env
+echo "CKEDITOR_ENVIRONMENT_ID=${CKEDITOR_ENVIRONMENT_ID}" >>.env
+echo "CKEDITOR_SECRET_KEY=${CKEDITOR_SECRET_KEY}" >>.env
+echo "CKEDITOR_UPLOAD_URL=${CKEDITOR_UPLOAD_URL}" >>.env
+echo "EDX_API_CLIENT_ID=${EDX_API_CLIENT_ID}" >>.env
+echo "EDX_API_CLIENT_SECRET=${EDX_API_CLIENT_SECRET}" >>.env
+echo "EMBEDLY_KEY=${EMBEDLY_KEY}" >>.env
+echo "MAILGUN_KEY=${MAILGUN_KEY}" >>.env
+echo "OCW_NEXT_SEARCH_WEBHOOK_KEY=${OCW_NEXT_SEARCH_WEBHOOK_KEY}" >>.env
+echo "OCW_WEBHOOK_KEY=${OCW_WEBHOOK_KEY}" >>.env
+echo "OLL_API_CLIENT_ID=${OLL_API_CLIENT_ID}" >>.env
+echo "OPENSEARCH_HTTP_AUTH=${OPENSEARCH_HTTP_AUTH}" >>.env
+echo "SECRET_KEY=${SECRET_KEY}" >>.env
```

Comment:
We shouldn't need the value from a deployed environment it could just be defined as a static string since it's only for local so commiting it shouldn't be an issue.